app/styles/toggle-switch.less

```diff
@@ -1,6 +1,15 @@
 .upf-toggle.upf-toggle--disabled {
   .upf-toggle__switch {
-    background-color: var(--color-gray-400);
+    background-color: var(--color-white);
```

Comment:
Why white background here? The switch has gray 400 background when it is disabled and not toggled, no?